packages/wonder-blocks-button/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -397,8 +392,9 @@ exports[`wonder-blocks-button example 3 1`] = `
     </span>
   </button>
   <a
+    aria-disabled="true"
```

Comment:
It's okay to have `disabled={true}` as well on buttons since you can still navigate to it via VO.  This means we'll be able to get rid fo that `e.preventDefault()` hack I added to the disabled click handlers.